Gems/Georeferencing/Code/Source/Tools/GeoreferencingEditorModule.cpp

```diff
@@ -0,0 +1,45 @@
+
+#include "GeoreferenceLevelEditorComponent.h"
+#include "GeoreferencingEditorSystemComponent.h"
+#include <Georeferencing/GeoreferencingTypeIds.h>
+#include <GeoreferencingModuleInterface.h>
+namespace Georeferencing
+{
+    class GeoreferencingEditorModule : public GeoreferencingModuleInterface
+    {
+    public:
+        AZ_RTTI(GeoreferencingEditorModule, GeoreferencingEditorModuleTypeId, GeoreferencingModuleInterface);
+        AZ_CLASS_ALLOCATOR(GeoreferencingEditorModule, AZ::SystemAllocator);
+
+        GeoreferencingEditorModule()
+        {
+            // Push results of [MyComponent]::CreateDescriptor() into m_descriptors here.
+            // Add ALL components descriptors associated with this gem to m_descriptors.
+            // This will associate the AzTypeInfo information for the components with the the SerializeContext, BehaviorContext and
+            // EditContext. This happens through the [MyComponent]::Reflect() function.
+            m_descriptors.insert(
+                m_descriptors.end(),
+                {
+                    GeoreferencingEditorSystemComponent::CreateDescriptor(),
+                    GeoReferenceLevelEditorComponent::CreateDescriptor(),
+                });
+        }
+
+        /**
+         * Add required SystemComponents to the SystemEntity.
+         * Non-SystemComponents should not be added here
+         */
+        AZ::ComponentTypeList GetRequiredSystemComponents() const override
+        {
+            return AZ::ComponentTypeList{
+                azrtti_typeid<GeoreferencingEditorSystemComponent>(),
+            };
+        }
+    };
+} // namespace Georeferencing
+
+#if defined(O3DE_GEM_NAME)
+AZ_DECLARE_MODULE_CLASS(AZ_JOIN(Gem_, O3DE_GEM_NAME, _Editor), Georeferencing::GeoreferencingEditorModule)
+#else
+AZ_DECLARE_MODULE_CLASS(Gem_Georeferencing_Editor, Georeferencing::GeoreferencingEditorModule)
+#endif
```

Comment:
I smell a monolithic build error here. Can you make sure that it will work with monolithic build?

I think the following change will be required:

```suggestion
AZ_DECLARE_MODULE_CLASS(Gem_Georeferencing_Editor, Georeferencing::GeoreferencingEditorModule)
```

Reply:
I was able to build release monolitic wiht following commands:
```
cmake -B build/linux_mono -S . -G "Ninja Multi-Config"  -DLY_MONOLITHIC_GAME=1
cmake --build build/linux_mono --target install --config release
```

Reply:
Thank you for testing it :+1: